actions/sequester/Quest2GitHub/Models/IssueExtensions.cs

```diff
@@ -31,11 +31,11 @@ month descending
     public static int? QuestStoryPoint(this StoryPointSize storyPointSize) =>
         storyPointSize.Size switch
         {
-            "🦔 Tiny" => 1,
-            "🐇 Small" => 3,
-            "🐂 Medium" => 5,
-            "🦑 Large" => 8,
-            "🐋 X-Large" => 13,
+            "🦔 Tiny (4h)" => 1,
+            "🐇 Small (1d)" => 3,
+            "🐂 Medium (3-5d)" => 5,
+            "🦑 Large (10d)" => 8,
+            "🐋 X-Large (20d)" => 13,
```

Comment:
I think it would be more resilient to change the switch to only match in the first N characters. That way, it works both if we've added the description in a project, and if we haven't.  Something like (only compiled in GFM, should be tested:

```csharp
storyPointSize.Size.Substring(0, 6) switch
        {
            "🦔 Tiny" => 1,
            "🐇 Smal" => 3,
            "🐂 Medi => 5,
            "🦑 Larg" => 8,
            "🐋 X-La" => 13,
```